src/components/Projects.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable max-len */
+/* eslint-disable no-unused-vars */
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components/macro';
+import FeaturedProject from './FeaturedProject';
+import OtherProject from './OtherProject';
+import { GridWrapper, SubHeading, StyledOther } from './ReusableStyles.style';
+import { featuredForDisplay, otherForDisplay } from '../utils/projectsForDisplay'
+
+const Projects = () => {
+  const [repos, setRepos] = useState([]);
+
+  useEffect(() => {
+    fetch('https://api.github.com/users/ElinSegelov/repos')
+      .then((data) => data.json())
+      .then((fetchedRepos) => setRepos(fetchedRepos))
+      .catch((error) => console.error(error))
+  }, []);
+
+  const featured = (repos.filter((repo) => featuredForDisplay.includes(repo.name)));
```

Comment:
You have done a fantastic job with the filtering and fetching the information from the API. If you would like to have new repos automatically coming in that are from Technigo, instead of `includes`, you could do the other way around and filter out the projects you do not want to show. Something like this `.filter(
				(repot) => (repo.name !== 'project-chatbot' || project.name !== 'project-guess-who' || project.name !== 'project-weather-app')
			)`